core/pom.xml

```diff
@@ -225,6 +225,16 @@
       <scope>test</scope>
     </dependency>
     <dependency>
+      <groupId>org.apache.ivy</groupId>
+      <artifactId>ivy</artifactId>
+      <version>${ivy.version}</version>
+    </dependency>
+    <dependency>
+      <groupId>oro</groupId>
```

Comment:
Ivy's pattern matcher uses it. Jenkins returned a ClassNotFoundException when that wasn't in.